WordPress/Classes/ViewRelated/Gutenberg/GutenbergNetworking.swift

```diff
@@ -24,13 +24,13 @@ struct GutenbergNetworkRequest {
     private func dotComRequest(with dotComID: NSNumber, completion: @escaping CompletionHandler) {
         blog.wordPressComRestApi()?.GET(dotComPath(with: dotComID), parameters: nil, success: { (response, httpResponse) in
             completion(.success(response))
-        }, failure: { (error, HTTPResponse) in
-            completion(.failure(error))
+        }, failure: { (error, httpResponse) in
+            completion(.failure(error.nsError(with: httpResponse)))
         })
     }
 
     private func dotComPath(with dotComID: NSNumber) -> String {
-        return path.replacingOccurrences(of: "/wp/v2/", with: "/wp/v2/sites/\(dotComID)/")
+        return path.replacingOccurrences(of: "/wp/v2/", with: "/wp/v2/sitesa/\(dotComID)/")
```

Comment:
Is this correct: `sitesa` ?

Reply:
Oh no! I changed that to test an error code. Didn't change it back 😬
Fixed now!